runner/README.md

```diff
@@ -0,0 +1,41 @@
+package runner is structs and interfaces to run Scoot processes.
+
+## Summary ##
+* Invoker runs one Command using Execer (and snapshot.Filer, etc.)
+* Controller controls when to Run something (using an Invoker)
+* Statuses queries Process history
+  * StatusesSync doesn't offer waiting, so requires client polling
+  * LegacyStatuses is the old Read API
+  * StatusWriter writes Process status (called by Controller)
+
+## Invoker (invoker.go) ##
+Invoker runs a Scoot command. This involves:
+* setup (check out a snapshot)
+* post-processing (save output)
+* babysitting (implement timeout and abort)
+* updates while running
+
+Conceptually, it lowers a Scoot Command from a high-level abstraction (Snapshots, etc.) to an Execer command (about directories that are paths in the local filesystem).
+
+## Controller (controller.go) ##
+Controller controls the Runs, starting a new one (Run()) or ending one (Abort()). It implements:
```

Comment:
IMHO - Controller should support running multiple commands concurrently in both daemon and cloud scoot
and yes Controller should support accepting a command even if there are no resources to run it immediately.